samples/dashboards/demo/climate/demo.css

```diff
@@ -184,6 +189,15 @@
     cursor: pointer;
 }
 
+#kpi-layout .highcharts-dashboards-component-title {
+    font-size: 1em !important;
```

Comment:
To be honest, I'm not convinced to use `!important`, I suggest using a more specific selector or inline style.